test/unit/dialects/mariadb/query-generator.test.js

```diff
@@ -750,9 +751,27 @@ if (dialect === 'mariadb') {
         {
           arguments: ['User'],
           expectation: 'SHOW INDEX FROM `User`',
-        }, {
+        },
+        {
+          arguments: [{ tableName: 'User', schema: 'schema' }],
+          expectation: 'SHOW INDEX FROM `schema`.`User`',
+        },
+        // FIXME: enable this test once fixed
```

Comment:
When would we fix this?

Reply:
Fixing it will like impact all query-generator methods so I want to do a follow up PR that fixes the issue everywhere, and adds tests for everything. Ideally before v7 is released, I would consider it to be a breaking change

There is also some overlap with #14634